### PR TITLE
The new EC/BC based model for GEM event selection has been added

### PR DESCRIPTION
**Pull request by sergueibaranov**

### CODE REVIEW

File: gemdaq-testing/gemsupervisor/src/common/GEMGLIBSupervisorWeb.cc [hunk 1]
```diff
@@ -411,7 +411,7 @@ void gem::supervisor::GEMGLIBSupervisorWeb::webTrigger(xgi::Input * in, xgi::Out
   hw_semaphore_.take();
 
   INFO(" webTrigger: sending L1A");
-  optohybridDevice_->SendL1A(1);
+  optohybridDevice_->SendL1A(100);
```
Comment (jsturdy) [inline gemdaq-testing/gemsupervisor/src/common/GEMGLIBSupervisorWeb.cc]: Please do not commit this change to the central pull request
In the "official" version, it should be SendL1A(1) such that it does exactly what the log message states

Comment (sergueibaranov) [inline gemdaq-testing/gemsupervisor/src/common/GEMGLIBSupervisorWeb.cc]: Ok. I'll update

Comment (sergueibaranov) [inline gemdaq-testing/gemsupervisor/src/common/GEMGLIBSupervisorWeb.cc]: Done

**Comment by jsturdy** [comment]
thanks!

**Pull request merged**